WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/DeleteDrinkViewModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ComparisonEngineUI.Data;
+using ComparisonEngineUI.Models;
+using System.Threading.Tasks;
+
+namespace ComparisonEngineUI.ViewModels
+{
+    class DeleteDrinkViewModel : BaseViewModel
+    {
+        private List<Bar> _barList { get; set; }
+
+        public List<Bar> BarList
+        {
+            get
+            {
+                return _barList;
+            }
+            set
+            {
+                if (value != _barList)
+                {
+                    _barList = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
+        public DeleteDrinkViewModel()
+        {
+            var restService = new RestService();
+            BarList = Task.Run(async () => await restService.GetData<List<Bar>>(Constants.BarsUrl)).Result;
+            
+        }
+    }
```

Comment:
As I see it, it should have both, user picks from which bar he wants to delete, then he picks which drinks in that bar he wants to delete(It would be weird if user just choose only one drinks, which would be deleted from all bars). The problem with this is having list of drinks corresponding to which bar was picked.